config/routes.rb

```diff
@@ -351,6 +351,7 @@
       collection do
         post 'add_existing_annotation'
         patch 'update_annotation'
+        post 'edit'
```

Comment:
I realized that some default routes (including `edit`) were created for `resources: annotations`. However, I thought maybe we'd like to explicitly state the routes we make use of so someone working on the project may use the routes file as a reference to see all list of all implemented routes. I can remove this if it is deemed unnecessary.